test/cloudflare_challenge_test.rb

```diff
@@ -21,6 +21,20 @@ def test_create_an_instance
     end
   end
 
+  def test_create_an_instance_with_custom_auth
+    ENV['CLOUDFLARE_EMAIL']   = nil
+    ENV['CLOUDFLARE_API_KEY'] = nil
+    VCR.use_cassette('acme-new-authz') do
+      a = CloudflareChallenge.new(zone: 'substrakt.com',
+                                  domains: ['www.substrakt.com', 'substrakt.com'],
+                                  api_key: 'fdhsufgdjshfgsd',
```

Comment:
Nope. Fake as hell.

VCR doesn't match on request bodies, so you're free to use fake data and it'll match it up to the live cassette.